storybookjs/storybook
# ADD integration test

automatic screenshot of cra-kitchen-sink and vue-kitchen-sink examples

This integrates into the `yarn test` workflow, and in fact is a jest project you can toggle on and off.
As a developer it's possible to just run all test in watch-mode all at once.
In the CI it's possible run all test or projects separately to get fine-grained feedback what's broken without having to look into the logs.

## What I did
- Added a jest project for integration testing
- Added a test for taking screenshots of a running example
- Integrated this into CI task

## How to test
- `yarn test --integration`
- change some implementation in cra or vue example that impacts welcome story significantly
- `yarn test --integration`
- should now fail and save diff screenshots
- `yarn test --integration --watch`
- press `u` to update, if implementation is correct

## Todo:
- [ ] Needs documentation in contributing.md or something
